apps/discord-bot/src/commands/gamecounts.command.ts

```diff
@@ -0,0 +1,72 @@
+import { INFO_COLOR } from '#constants';
+import { ApiService, Page, PaginateService } from '#services';
+import { Command, CommandContext, EmbedBuilder } from '@statsify/discord';
+import { prettify } from '@statsify/util';
+
+@Command({
+  description: (t) => t('commands.gameCounts'),
+})
+export class GameCountsCommand {
```

Comment:
It would be cool if the dropdown had emojis for the options. I think you would just have to edit `PaginateService` and some if its types.